scaladoc/src/dotty/tools/scaladoc/snippets/SnippetChecker.scala

```diff
@@ -8,11 +8,12 @@ import java.io.File
 class SnippetChecker(val classpath: String, val tastyDirs: Seq[File]):
   private val sep = System.getProperty("path.separator")
   private val cp = List(
-    System.getProperty("java.class.path"),
-    Paths.get(classpath).toAbsolutePath,
-    tastyDirs.map(_.getAbsolutePath()).mkString(sep)
+    tastyDirs.map(_.getAbsolutePath()).mkString(sep),
+    classpath,
+    System.getProperty("java.class.path")
```

Comment:
I am not convinced that this is a good idea. If this is run from sbt will we get the sbt jars on classpath?

Reply:
I need to check that, maybe I will try to import flexmark during scaladoc self documenting

Reply:
I've checked and apparently all dependencies from sbt are also on classpath. Additionally, we've got `classpath` argument in scaladoc which should lead to all dependencies needed by documented project.

Reply:
This is a thing we need to discuss